internal/quickstart/choose_sdk.go

```diff
@@ -108,35 +108,36 @@ func (s sdkDetail) FilterValue() string { return "" }
 
 var SDKs = []sdkDetail{
 	{canonicalName: "react", displayName: "React", kind: clientSideSDK, hasInstructions: true},
-	{canonicalName: "node-server", displayName: "Node.js (server-side)", kind: serverSideSDK},
+	{canonicalName: "node-server", displayName: "Node.js (server-side)", kind: serverSideSDK, hasInstructions: true},
```

Comment:
I'll add a follow up PR to remove this extra check now that we're no longer fetching these from the README's